apps/extension/src/core/media/openrouter.ts

```diff
@@ -33,7 +33,15 @@ export function init(
       },
       transformResponse: (res) => {
         const anyRes = res as any
-        return anyRes["data"].map(
+        // TODO: remove this once openrouter endpoint is migrated from returning a "data" param
+        const result = "data" in anyRes ? anyRes.data : anyRes
+        let objects: any = []
```

Comment:
couldn't this be simplified to `const objects = Array.isArray(result) ? result : result.objects`? since objects is the only possible key name in the new api?

also, weren't we going to name it `generations` not `objects`? (maybe `results` would be even better / more applicable to predictive models)